cmd/util/ledger/migrations/cadence_values_migration.go

```diff
@@ -384,20 +386,32 @@ func NewCadence1CapabilityValueMigration(
 
 	return &CadenceBaseMigration{
 		name:                              "cadence_capability_value_migration",
-		reporter:                          rwf.ReportWriter("cadence-capability-value-migration"),
+		reporter:                          rwf.ReportWriter(capabilityValueMigrationReporterName),
 		diffReporter:                      diffReporter,
 		logVerboseDiff:                    opts.LogVerboseDiff,
 		verboseErrorOutput:                opts.VerboseErrorOutput,
 		checkStorageHealthBeforeMigration: opts.CheckStorageHealthBeforeMigration,
 		valueMigrations: func(
 			_ *interpreter.Interpreter,
-			_ environment.Accounts,
+			accounts environment.Accounts,
 			reporter *cadenceValueMigrationReporter,
 		) []migrations.ValueMigration {
+
+			idGenerator := environment.NewAccountLocalIDGenerator(
+				tracing.NewMockTracerSpan(),
+				util.NopMeter{},
+				accounts,
```

Comment:
Not sure if thi `idGenerator` needs to be shared between this (capability) migration and the link value migration.

Reply:
@onflow/flow-cadence-execution